ystia/yorc
# Feature/gh 723 a4 c 3 2 0

# Pull Request description

## Description of the change

Updated the bootstrap to install Alien4Cloud 3.2.0 and its dependency JDK 15.

### What I did

`versions.yaml`
Updated Alien4Cloud version

`commands/bootstrap/inputs.go`
Updated Alien4Cloud default URL and OpenJDK default URL as Alien4Cloud 3.2.0 expects now JDK 15.

`commands/bootstrap/resources/topology/tosca_types.zip`
Misc.: fixed a pip version issue installing yorc dependencies on CentOS 7 found while validating the bootstrap. 

### How to verify it

Bootstrap yorc, keeping bootstrap default value for Alien4Cloud URL.
Check the bootstrapped setup runs Alien4Cloud 3.2.0

### Description for the changelog

Support Alien4Cloud 3.2.0 ([GH-723](https://github.com/ystia/yorc/issues/723))

## Applicable Issues

Fixes #723
